tv-shows/src/components/features/shows/ShowContainer/ShowContainer.tsx

```diff
@@ -3,32 +3,36 @@ import { ShowReviewSection } from "@/components/features/shows/ShowReviewSection
 import { IReviewItem, IReviewList } from "@/typings/Review";
 import { Fragment, useEffect, useState } from "react";
 import { IShow } from "@/typings/Show";
+import { Flex } from "@chakra-ui/react";
+import { useParams } from "next/navigation";
 
-const mockShow = {
-    title: 'Dark',
-    description: 'A family saga with a supernatural twist, set in a German town where the disappearance of two young children exposes the relationships among four families.',
-    averageRating: 5,
-    imageUrl: 'https://dark.netflix.io/share/global.png'
+const allReviewList : IReviewList = {
+    reviews: []
 };
 
-export default function ShowContainer() {
+interface IShowContainerProps {
+    showProp: IShow;
+}
+
+export default function ShowContainer({ showProp } : IShowContainerProps) {
     useEffect(() => {
         const loadedList = loadFromLocalStorage();
         updateAverageRating(loadedList);
         setReviewList(loadedList);
     }, []);
 
-    const [reviewList, setReviewList] = useState([]);
-    const [show, setShow] = useState(mockShow);
+    const [reviewList, setReviewList] = useState(allReviewList);
+    const [show, setShow] = useState(showProp);
+    const params = useParams();
 
     const saveToLocalStorage = (reviewList: IReviewList) => {
-        localStorage.setItem('reviewList', JSON.stringify(reviewList));
+        localStorage.setItem(`reviewList-${params.id}`, JSON.stringify(reviewList));
```

Comment:
Ovdje smo rekli da bi baš u review upisivali showId na koji se taj review kači. Ovako bi se nakotilo previše localStorage itema.
Ali sad ćemo to zanemarit 🔥